keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/verification/VerificationRepository.kt

```diff
@@ -34,7 +34,7 @@ interface VerificationRepository {
     context: VerificationContext,
     verification: Verification,
     status: VerificationStatus,
-    metadata: Map<String, Any?>? = null
+    metadata: Map<String, Any?> = emptyMap()
```

Comment:
made this parameter non-null so it doesn't get propagated to the database as `null`

Reply:
Comment at line 31 also needs updating. (Alas, GH doesn't let me annotate non-changed lines)

Reply:
Okay I changed it